tcf_website/management/commands/import_csv_review_drive.py

```diff
@@ -0,0 +1,227 @@
+            help=(
+                "Filepath of csv to upload in /review_drive_responses/"
+                "Must be in correct format."
+            ),
+            type=str,
+        )
+        parser.add_argument(
+            "season",
+            help=(
+                "What season the reviews took place i.e. Fall, January, Spring, Summer"
+            ),
+            type=str,
+        )
+        parser.add_argument(
+            "year",
+            help="Year of reviews i.e. 2024",
+            type=int,
+        )
+
+        # Named (optional) arguments
+        parser.add_argument(
+            "--verbose",
+            action="store_true",
+            help="Verbose output",
+        )
+
+    def handle(self, *args, **options):
+        verbose = options["verbose"]
+        filename = options["filename"]
+        season = options["season"]
+        year = options["year"]
+
+        # Get semester object from args
+        semester = Semester.objects.filter(
+            season=season.upper(), year=year
+        ).first()
+        if semester is None:
+            print("ERROR: Semester not found")
+            return
+
+        # Needs to be created ahead of time in the db
+        dummy_account = User.objects.filter(computing_id__iexact=settings.REVIEW_DRIVE_ID).first()
+        if dummy_account is None:
+            dummy_account = User.objects.create_user(computing_id=settings.REVIEW_DRIVE_ID,
+                                                     graduation_year=year,
+                                                     username=settings.REVIEW_DRIVE_ID,
+                                                     password=settings.REVIEW_DRIVE_PASSWORD,
+                                                     email=settings.REVIEW_DRIVE_EMAIL,
+                                                     first_name='Review',
+                                                     last_name='Drive')
+
+        print("Starting file upload...")
+        create_reviews(verbose, filename, semester, dummy_account)
+
+
+def create_reviews(verbose, filename, semester, dummy_account):
+    """Creates reviews based on CSV info."""
+    with open(os.path.join(DATA_DIR, filename), mode="r", encoding='utf-8') as file:
+        csv_file = csv.reader(file)
+
+        for i, line in enumerate(csv_file):
+            if i == 0:
+                continue
+
+            # Strip needed to remove random whitespace for exact match
+            email = line[1].strip()
+            mnemonic = line[2].strip()
+            num = int(line[3].strip())
+            instructor_arr = line[4].strip().split(" ", 1)
+
+            if verbose:
+                print(email, mnemonic, num, instructor_arr)
+
+            # Use account if found. Otherwise, use dummy user
+            account = User.objects.filter(email=email).first()
+            if account is not None and verbose:
+                print("Account already found for", email)
+            elif account is None:
```

Comment:
Here, we can just set `email = ""`, then use `email` instead of `review_email`.

Reply:
Changed it to use a ternary if statement later on instead of review_email.